src/library_webgl.js

```diff
@@ -3794,19 +3794,47 @@ for (/**@suppress{duplicate}*/var i = 0; i <= {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
   glDrawElements: (mode, count, type, indices) => {
 #if FULL_ES2
     var buf;
+    var vertexes = 0;
     if (!GLctx.currentElementArrayBufferBinding) {
       var size = GL.calcBufLength(1, type, 0, count);
       buf = GL.getTempIndexBuffer(size);
       GLctx.bindBuffer(0x8893 /*GL_ELEMENT_ARRAY_BUFFER*/, buf);
       GLctx.bufferSubData(0x8893 /*GL_ELEMENT_ARRAY_BUFFER*/,
                           0,
                           HEAPU8.subarray(indices, indices + size));
+      
+      // Calculating vertex count if shader's attribute data is on client side
+      if (count > 0) {
+        for (var i = 0; i < GL.currentContext.maxVertexAttribs; ++i) {
+          var cb = GL.currentContext.clientBuffers[i];
+          if (cb.clientside && cb.enabled) {
+            const arrayClasses = {
+                0x1401 /* GL_UNSIGNED_BYTE */: Uint8Array,
+                0x1403 /* GL_UNSIGNED_SHORT */: Uint16Array,
+#if FULL_ES3
+                0x1405 /* GL_UNSIGNED_INT */: Uint32Array
+#endif
+            };
+
+            const arrayClass = arrayClasses[type];
+            if (!arrayClass) {
+              GL.recordError(0x502 /* GL_INVALID_OPERATION */);
+              err('type is not supported in glDrawElements');
```

Comment:
This logging should be guarded by `#if GL_ASSERTIONS`, see e.g. line 336.

Reply:
Changed.